### ISSUE TITLE
[Feature Request] Long Click for Detailsview

### ISSUE THREAD

**Issue by newhinton** [FredJul/Flym#463]
I developed the feature where you can directly open an feedentry in your browser. I want to extend this so that you can long-click to open the internal view of flym. I got it working that there is a long click action (which prints the url to logcat) but sadly i cannot get the preference or the activity to start. Can you help me out?

**Comment by FredJul** [comment]
Thank you for trying to help, PR are always welcome.

However, I'm not sure to understand what you try to achieve: I agree that a long click to open on browser would be a good idea. But what is the "long click to open the internal view of flym" ? Isn't what the simple tap is doing ?

**Comment by newhinton** [comment]
yes, exactly. But some time ago i opened a pull request ( #416) which allows to open an entry with a single tap. I want to expand that, so that if a single tap opens the browser, a long press opens the default, internal view.

That is useful incase the browser does not work because you do not have internet and want to access the cached version from flym

**Comment by FredJul** [comment]
Sorry for the late answer. I better understand what you want to do. You juste need to call:
`navigator.goToEntryDetails(entry.id, allEntryIds)`  in order to display one entry.
